backend/src/main/java/gov/cdc/usds/simplereport/api/model/filerow/TestResultRow.java

```diff
@@ -122,6 +126,92 @@ public class TestResultRow implements FileRow {
   static final String TESTING_LAB_CITY = "testing_lab_city";
   static final String TESTING_LAB_STATE = "testing_lab_state";
   static final String TESTING_LAB_ZIP_CODE_FIELD = "testing_lab_zip_code";
+  static Set<String> fluOnlyTestPerformedLoinc =
+      Set.of(
+          "100973-7",
```

Comment:
Are these LOINCs ever represented in documentation or guidance without the dash `-`? Wondering if we would ever want to accept LOINCs in a different format.

Reply:
Great observation, the idea was to add the validation in place and then update the docs once we are ready to release it